value/src/main/java/com/google/auto/value/processor/BuilderSpec.java

```diff
@@ -131,10 +132,10 @@
     ImmutableSet<ExecutableElement> toBuilderMethods(
         Types typeUtils, Set<ExecutableElement> abstractMethods) {
 
-      ImmutableList<String> builderTypeParamNames =
-          FluentIterable.from(builderTypeElement.getTypeParameters())
-              .transform(SimpleNameFunction.INSTANCE)
-              .toList();
+      List<String> builderTypeParamNames =
+          builderTypeElement.getTypeParameters().stream()
+              .map(e -> e.getSimpleName().toString())
+              .collect(toList());
```

Comment:
Yes, I think I will do another pass over the code replacing for-loops with streams where appropriate.